examples/dns/tracer.js

```diff
@@ -36,5 +28,18 @@ module.exports = (serviceName) => {
   // Initialize the OpenTelemetry APIs to use the NodeTracerProvider bindings
   provider.register();
 
+  registerInstrumentations({
+    instrumentations: [{
+      plugins: {
+        dns: {
+          enabled: true,
+          path: '@opentelemetry/plugin-dns',
```

Comment:
There is https://github.com/open-telemetry/opentelemetry-js-contrib/pull/358 to move to instrumentation. Would be good to integrate it.